pkg/api/controller.go

```diff
@@ -2107,6 +2107,150 @@ func (c *Controller) ResetBranch(w http.ResponseWriter, r *http.Request, body Re
 	writeResponse(w, r, http.StatusNoContent, nil)
 }
 
+func (c *Controller) ImportStart(w http.ResponseWriter, r *http.Request, body ImportStartJSONRequestBody, repository, branch string) {
+	if !c.authorize(w, r, permissions.Node{
```

Comment:
I think its redundant in this implementation - since the user isn't calling the CreateMetaRange API explicitly.
IMO we should be good with just the Commit permission

Reply:
Agree - looks we have permissions in different levels, when we could use commit for write metarange or use different user level permission to capture range/metarange/commit/log changes.